avaframe/com1DFA/com1DFA.py

```diff
@@ -651,9 +681,17 @@ def initializeSimulation(cfg, demOri, inputSimLines, logName, relThField=''):
     # initialize simulation
     # create primary release area particles and fields
     releaseLine['header'] = demOri['header']
-    particles = initializeParticles(cfgGen, releaseLine, dem, logName=logName, relThField=relThField)
+    inputSimLines['releaseLine']['header'] = demOri['header']
+    particles = initializeParticles(cfgGen, releaseLine, dem, inputSimLines=inputSimLines, logName=logName, relThField=relThField)
     particles, fields = initializeFields(cfgGen, dem, particles)
 
+    # perform initialisation step for redistributing particles
+    if cfg['GENERAL'].getboolean('iniStep') and not cfg['GENERAL'].getboolean('initialiseParticlesFromFile'):
```

Comment:
Should this be checked in advance with an error if it does not fit?

Reply:
done